src/language-js/embed/html.js

```diff
@@ -93,4 +93,5 @@ async function format(path, print, textToDoc, options, { parser }) {
   ]);
 }
 
-export default format;
+export const formatHtml = (textToDoc, print, path, options) => format(textToDoc, print, path, options, { parser: "html" })
+export const formatAngular = (textToDoc, print, path, options) => format(textToDoc, print, path, options, { parser: "angular" })
```

Comment:
No need to wrap the `parser` string in a object here.

Also, if we make it the first parameter, it'll be possible to rewrite these 2 functions this way:
```js
export const formatAngular = format.bind(null, "angular");
```